examples/issue_labels/terraform.tf

```diff
@@ -1,10 +1,10 @@
 terraform {
   # see https://www.terraform.io/docs/language/settings/index.html#specifying-provider-requirements
   required_providers {
-    # see https://registry.terraform.io/providers/integrations/github/5.2.0
+    # see https://registry.terraform.io/providers/integrations/github/5.3.0
     github = {
       source  = "integrations/github"
-      version = "5.2.0"
+      version = ">= 5.3.0, < 6.0.0"
```

Comment:
This is great but I think  you can get a simpler syntax with pessimistic version constraints.
This will have the same result as you have here.
Not a big deal, but just passing along :)

Example: "~>5.3"

https://developer.hashicorp.com/terraform/language/expressions/version-constraints#-3